modules/persistent-storage-local-install.adoc

```diff
@@ -49,3 +49,74 @@ $ oc annotate project local-storage openshift.io/node-selector=''
 .. Click *Subscribe*.
 
 . Once finished, the Local Storage Operator will be listed in the *Installed Operators* section of the web console.
+
+.. Navigate to *Operators* -> *OperatorHub*.
+
+.. Type *Local Storage* into the filter box to locate the Local Storage Operator.
+
+.. Click *Install*.
+
+.. On the *Create Operator Subscription* page, select *A specific namespace on the cluster*. Select *local-storage* from the drop-down menu.
+
+.. Adjust the values for the *Update Channel* and *Approval Strategy* to the desired values.
+
+.. Click *Subscribe*.
+
+. Once finished, the Local Storage Operator will be listed in the *Installed Operators* section of the web console.
+
+.From the CLI
+. Install the Local Storage Operator from the CLI.
+
+.. Create a Subscription object YAML file to subscribe a namespace to the Local Storage Operator,
+such as `local-storage.yaml`:
++
+.Example Subscription
+[source,yaml]
+[source,yaml]
+----
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: local-storage-operator
+    namespace: openshift-operators <1>
+  spec:
+    channel: alpha
+    name: local-storage-operator <2>
+    source: redhat-operators <3>
+    sourceNamespace: openshift-marketplace <4>
+----
+<1> For `AllNamespaces` InstallMode usage, specify the `openshift-operators`
+namespace. Otherwise, specify the relevant single namespace for
+`SingleNamespace` InstallMode usage.
+<2> Name of the Operator to subscribe to.
+<3> Name of the CatalogSource that provides the Operator.
+<4> Namespace of the CatalogSource. Use `openshift-marketplace` for the default
+OperatorHub CatalogSources.
+
+. Create the Local Storage Operator object by entering the following command:
++
+----
+$ oc apply -f local-storage.yaml
+----
++
+At this point, the Operator Lifecycle Manager (OLM) is now aware of the Local Storage Operator. A ClusterServiceVersion (CSV) for the Operator should appear in the target namespace, and APIs provided by the Operator should be available for creation.
++
+. Verify local storage installation by checking that all Pods and the Local Storage Operator have been created:
+
+.. Check that all the required Pods have been created:
```

Comment:
Not sure if this step (`oc get pods`) is really necessary for verifying that LSO is working?

Reply:
keep this is ok for me.